posthog-rails/lib/posthog/rails/active_job.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'posthog/rails/parameter_filter'
+
+module PostHog
+  module Rails
+    # ActiveJob integration to capture exceptions from background jobs
+    module ActiveJobExtensions
+      include ParameterFilter
+
+      def perform_now
+        super
+      rescue StandardError => e
+        # Capture the exception with job context
+        capture_job_exception(e)
+        raise
+      end
+
+      private
+
+      def capture_job_exception(exception)
+        return unless PostHog.rails_config&.auto_instrument_active_job
```

Comment:
Hmmm, I'd much rather this be `PostHog::Rails.config` across the whole PR. Can you update this, please?